setup.py

```diff
@@ -124,7 +124,11 @@
     'build': BUILD_DEPS,
 }
 
-EXT_CFLAGS = ['-O2']
+EXT_CFLAGS: List[str] = []
+if flag := os.getenv('EDGEDB_OPT_CFLAG'):
```

Comment:
We already have `EDGEDB_DEBUG` env var down below that affects Cython compilation, perhaps we can reuse that?

Reply:
Setting that actually slows down compilation, though, from 14s to 20s relative to just passing -O0